curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

```diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+In which scenario is using an attribute selector a bad idea?
 
 #### --distractors--
 
-Placeholder distractor 1
+```css
+input[type="text"]:focus {
+    background-color: yellow;
+}
 
----
-
-Placeholder distractor 2
+```
 
 ---
+```css
+img[alt] {
+    border: 2px solid red;
+}
+```
```

Comment:
````suggestion
---

```css
img[alt] {
    border: 2px solid red;
}
```
````

I'm trying my best but I am really confused by what happened to the diff of the file